docs/Concepts/Privacy/PrivacyEnhancements.md

```diff
@@ -0,0 +1,129 @@
+
+## Privacy Enhanced Transaction End to End Flow
+
+Refer [here](PrivateTransactionLifecycle.md) for end to end flow of 'standard' private transaction.
+
+![](../../images/PrivacyEnhancements.png)
+
+In this example we walk through the flow of a private transaction on a 'privacy enhanced contract' between Nodes A & B.
+
+1. User pushing a private transaction from Node A private for Node B
+
+    - The transaction payload will include the `PrivacyFlag` with value `1` for PP and `3` for PSV contract
+
+2. Node A Quorum reading the `PrivacyFlag` runs an EVM transaction simulation to gather all the affected contracts and the ACOTH(s) associated to contract accounts. For PSV transactions, it also calculates an execution hash (merkle root) from all the affected contracts resulting from the transaction simulation.
+
+3. Node A Quorum pushes the transaction payload, `PrivacyFlag`, ACOTHs (& the merkle root for `PSV`) to Node A Tessera.
+
+4. Node A Tessera generates proofs (secure hashes) for the ACOTHs and use them to validate that the originating party has access to all relevant transactions. In addition for `PSV` it would also verify the participants list against the list in each of the ACOTH transactions (as in `PSV` transactions the recipient list is shared across all nodes party to the transaction). If the list doesn't match it will return failure on `/send` to Node A Quorum.
+
+5. Node A Tessera pushes to Node B Tessera the encrypted payload, ACOTH <-> proofs (secure hash) mappings (for `PSV` transaction it will also push the `privateFor` list and merkle root).
+
+6. Node B Tessera computes and compares proofs (secure hash) from Node A Tessera (for `PSV` it also verifies participants list of ACOTHs against the `privateFor` list).
+
+7. Node B Tessera returns a SUCCESS response to Node A Tessera - even if the compute and compare mismatched (in order to prevent Node A finding out the recipient list of a contract/transaction) but it will not store the payload/ACOTH<->Securehash mapping based on the outcome.
+
+8. Node A Tessera returns the hash for the encrypted transaction payload to Node A Quorum
+
+9. Node A mints the transaction across the network.
+
+10. Node A & Node B being party to the contract will `/receive` decrypted payload, ACOTHs (for `PSV` also merkle root) from the respective Tessera Nodes.
+
+11. Both Nodes execute the transaction and compare the ACOTHs (and execution hash for `PSV`) and update the transaction receipt accordingly to mark transaction execution completion.
+
+**Note : If the EVM simulation impacts more than one contract, all contracts should have the same `PrivacyFlag`, otherwise the transaction is rejected. All contracts ACOTHs are included in the transmission and the Tessera node will create individual proofs (secure hash) for each ACOTH.**
```

Comment:
Add all abbreviations definitions at the end of the doc using the https://python-markdown.github.io/extensions/abbreviations/ syntax.

Use the `!!!note` admonition for notes.

```suggestion
!!! note
    If the EVM simulation impacts more than one contract, all contracts should have the same `PrivacyFlag`, otherwise the transaction is rejected.
    All contracts ACOTHs are included in the transmission and the Tessera node will create individual proofs (secure hash) for each ACOTH.**

*[PP]: Counter Party Protection
*[PSV]: Private State Validation
*[ACOTH]: Affected Contract's Original Transaction's encrypted payload Hash
*[SP]: Standard Private
*[EVM]: Ethereum Virtual Machine
*[P2P]: Peer to peer
```